README.md

```diff
@@ -22,7 +22,7 @@ Wagtail-TreeModelAdmin is an extension for Wagtail's [ModelAdmin](http://docs.wa
 
 - Python 3.8+
 - Django 3.2 (LTS), 4.1 (current)
-- Wagtail 4.0+
+- Wagtail 5+,<5.2
```

Comment:
Shouldn't this include 4.2 if we're maintaining support for now?

Reply:
Fixed in 5f84cd0